<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MP3EDJ</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/novaPredpona/Tuziibanez-Profesional-Red-Mp3-file.ico</normaloff>:/novaPredpona/Tuziibanez-Profesional-Red-Mp3-file.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page"/>
        <widget class="QWidget" name="page_3">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout">
              <item row="3" column="1" colspan="2">
               <widget class="QListWidget" name="listWidget"/>
              </item>
              <item row="1" column="2">
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="Phonon::SeekSlider" name="seekSlider" native="true">
                  <property name="maximumSize">
                   <size>
                    <width>16777099</width>
                    <height>16777178</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="2">
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="Izvajalec">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="Naslov">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="BPM">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="Key">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLCDNumber" name="lcdNumber">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>70</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>9</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="mouseTracking">
                   <bool>false</bool>
                  </property>
                  <property name="autoFillBackground">
                   <bool>true</bool>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(18, 28, 45);</string>
                  </property>
                  <property name="smallDecimalPoint">
                   <bool>false</bool>
                  </property>
                  <property name="segmentStyle">
                   <enum>QLCDNumber::Flat</enum>
                  </property>
                  <property name="value" stdset="0">
                   <double>0.000000000000000</double>
                  </property>
                  <property name="intValue" stdset="0">
                   <number>0</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="1">
               <widget class="QCheckBox" name="checkBoxBrisi">
                <property name="text">
                 <string>DELETE</string>
                </property>
               </widget>
              </item>
              <item row="3" column="3">
               <widget class="Phonon::VolumeSlider" name="volumeSlider" native="true">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>200</height>
                 </size>
                </property>
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2"/>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="statusTip">
         <string extracomment="jk"/>
        </property>
        <property name="accessibleDescription">
         <string/>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="textDirection">
         <enum>QProgressBar::BottomToTop</enum>
        </property>
        <property name="format">
         <string extracomment="komentar">%p%</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1260</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMP3EDJ">
    <property name="tearOffEnabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>MP3EDJ</string>
    </property>
    <addaction name="actionAdd_Files"/>
    <addaction name="actionAdd_Folder"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
    <addaction name="separator"/>
    <addaction name="actionEXIT"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionSoftware"/>
    <addaction name="actionDeveloper"/>
    <addaction name="separator"/>
    <addaction name="actionWebpage"/>
   </widget>
   <widget class="QMenu" name="menuDirectory">
    <property name="title">
     <string>Editor</string>
    </property>
    <addaction name="actionMove_all_files_to_root_folder"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_BPM_Tags_to_Files"/>
    <addaction name="actionAdd_KEY_Tags_to_Files"/>
    <addaction name="separator"/>
    <addaction name="actionMove_Files_to_BPM_Sub_Folders"/>
    <addaction name="actionMove_Files_to_KEY_Sub_Folders"/>
    <addaction name="separator"/>
    <addaction name="actionRename_Files_by_ID3v2_Data"/>
   </widget>
   <widget class="QMenu" name="menuPlayer">
    <property name="title">
     <string>Player</string>
    </property>
    <addaction name="actionPrev"/>
    <addaction name="actionPlay"/>
    <addaction name="actionNext"/>
    <addaction name="actionPause"/>
   </widget>
   <addaction name="menuMP3EDJ"/>
   <addaction name="menuDirectory"/>
   <addaction name="menuPlayer"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="ToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolTip">
    <string>uredi playliste</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMusic"/>
   <addaction name="actionGenre"/>
   <addaction name="actionConverter"/>
   <addaction name="actionStretch"/>
   <addaction name="actionPlaylist"/>
   <addaction name="actionSplitter"/>
   <addaction name="actionNormalize"/>
   <addaction name="actionRecorder"/>
   <addaction name="separator"/>
   <addaction name="actionEXIT"/>
  </widget>
  <widget class="QToolBar" name="PlayBar">
   <property name="windowTitle">
    <string>PlayBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPrev"/>
   <addaction name="actionPlay"/>
   <addaction name="actionPause"/>
   <addaction name="actionNext"/>
  </widget>
  <widget class="QToolBar" name="MoveBar">
   <property name="windowTitle">
    <string>MoveBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDelete"/>
   <addaction name="actionTECHNO"/>
   <addaction name="actionHardTechno"/>
   <addaction name="actionHardCore"/>
   <addaction name="actionDrum_Bass"/>
  </widget>
  <action name="actionMusic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/novaPredpona/Tuziibanez-Profesional-Red-Audio-folder.ico</normaloff>:/novaPredpona/Tuziibanez-Profesional-Red-Audio-folder.ico</iconset>
   </property>
   <property name="text">
    <string>Music</string>
   </property>
   <property name="statusTip">
    <string>Sort files by genre</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionGenre">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/novaPredpona/Vargas21-Aquave-Metal-Music.ico</normaloff>:/novaPredpona/Vargas21-Aquave-Metal-Music.ico</iconset>
   </property>
   <property name="text">
    <string>Genre</string>
   </property>
  </action>
  <action name="actionPlaylist">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/novaPredpona/Franksouza183-Fs-Mimetypes-playlist.ico</normaloff>:/novaPredpona/Franksouza183-Fs-Mimetypes-playlist.ico</iconset>
   </property>
   <property name="text">
    <string>Playlist</string>
   </property>
  </action>
  <action name="actionStretch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/novaPredpona/Fatcow-Farm-Fresh-Application-side-expand.ico</normaloff>:/novaPredpona/Fatcow-Farm-Fresh-Application-side-expand.ico</iconset>
   </property>
   <property name="text">
    <string>Stretch</string>
   </property>
  </action>
  <action name="actionRecorder">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/novaPredpona/Capital18-Capital-Suite-Drives-CD-Drive.ico</normaloff>:/novaPredpona/Capital18-Capital-Suite-Drives-CD-Drive.ico</iconset>
   </property>
   <property name="text">
    <string>Recorder</string>
   </property>
  </action>
  <action name="actionSplitter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/novaPredpona/Semlabs-Web-Blog-Arrow-split-right.ico</normaloff>:/novaPredpona/Semlabs-Web-Blog-Arrow-split-right.ico</iconset>
   </property>
   <property name="text">
    <string>Splitter</string>
   </property>
  </action>
  <action name="actionEXIT">
   <property name="icon">
    <iconset>
     <normaloff>:/novaPredpona/Tuziibanez-Profesional-Red-Turn-off.ico</normaloff>:/novaPredpona/Tuziibanez-Profesional-Red-Turn-off.ico</iconset>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </action>
  <action name="actionSoftware">
   <property name="text">
    <string>Software</string>
   </property>
  </action>
  <action name="actionDeveloper">
   <property name="text">
    <string>Developer</string>
   </property>
  </action>
  <action name="actionWebpage">
   <property name="text">
    <string>Webpage</string>
   </property>
  </action>
  <action name="actionNormalize">
   <property name="icon">
    <iconset>
     <normaloff>:/novaPredpona/Tuziibanez-Profesional-Red-Volume.ico</normaloff>:/novaPredpona/Tuziibanez-Profesional-Red-Volume.ico</iconset>
   </property>
   <property name="text">
    <string>Normalize</string>
   </property>
  </action>
  <action name="actionConverter">
   <property name="icon">
    <iconset>
     <normaloff>:/novaPredpona/Custom-Icon-Design-Office-Export.ico</normaloff>:/novaPredpona/Custom-Icon-Design-Office-Export.ico</iconset>
   </property>
   <property name="text">
    <string>Converter</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset>
     <normaloff>:/novaPredpona/Ahmadhania-Spherical-Play.ico</normaloff>:/novaPredpona/Ahmadhania-Spherical-Play.ico</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset>
     <normaloff>:/novaPredpona/Ahmadhania-Spherical-Stop.ico</normaloff>:/novaPredpona/Ahmadhania-Spherical-Stop.ico</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset>
     <normaloff>:/novaPredpona/Ahmadhania-Spherical-Next-track.ico</normaloff>:/novaPredpona/Ahmadhania-Spherical-Next-track.ico</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionPrev">
   <property name="icon">
    <iconset>
     <normaloff>:/novaPredpona/Ahmadhania-Spherical-Previous-track.ico</normaloff>:/novaPredpona/Ahmadhania-Spherical-Previous-track.ico</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="softKeyRole">
    <enum>QAction::NoSoftKey</enum>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset>
     <normaloff>:/novaPredpona/Creative-Freedom-Shimmer-Document-Delete.ico</normaloff>:/novaPredpona/Creative-Freedom-Shimmer-Document-Delete.ico</iconset>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionTECHNO">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>TECHNO</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionHardTechno">
   <property name="text">
    <string>HardTechno</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionHardCore">
   <property name="text">
    <string>HardCore</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="actionDrum_Bass">
   <property name="text">
    <string>Drum&amp;Bass</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </action>
  <action name="actionAdd_Folder">
   <property name="icon">
    <iconset>
     <normaloff>:/novaPredpona/Vargas21-Aquave-Metal-Music.ico</normaloff>:/novaPredpona/Vargas21-Aquave-Metal-Music.ico</iconset>
   </property>
   <property name="text">
    <string>Add Folder</string>
   </property>
  </action>
  <action name="actionAdd_Files">
   <property name="text">
    <string>Add Files</string>
   </property>
  </action>
  <action name="actionMove_all_files_to_root_folder">
   <property name="text">
    <string>Move ALL Files to root Folder</string>
   </property>
  </action>
  <action name="actionAdd_BPM_Tags_to_Files">
   <property name="text">
    <string>Add BPM Tags to Files</string>
   </property>
  </action>
  <action name="actionAdd_KEY_Tags_to_Files">
   <property name="text">
    <string>Add KEY Tags to Files</string>
   </property>
  </action>
  <action name="actionMove_Files_to_BPM_Sub_Folders">
   <property name="text">
    <string>Move Files to BPM Sub Folders</string>
   </property>
  </action>
  <action name="actionMove_Files_to_KEY_Sub_Folders">
   <property name="text">
    <string>Move Files to KEY Sub Folders</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionRename_Files_by_ID3v2_Data">
   <property name="text">
    <string>Rename Files by ID3v2 Data</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
  <include location="play.qrc"/>
 </resources>
 <connections/>
</ui>
